doc/src/tutorials/biomechanics/biomechanics.rst

```diff
@@ -0,0 +1,799 @@
+   ...
+   ...         """
+   ...         return self.radius*self.coordinate.diff(me.dynamicsymbols._t)
+   ...
+   ...     def compute_loads(self, force_magnitude):
+   ...         """Loads in the correct format to be supplied to `KanesMethod`.
+   ...
+   ...         Forces applied to origin, insertion, and P from the muscle wrapped
+   ...         over circular arc of radius r.
+   ...
+   ...         """
+   ...
+   ...         parent_tangency_point = me.Point('Aw')  # fixed in parent
+   ...         child_tangency_point = me.Point('Bw')  # fixed in child
+   ...
+   ...         parent_tangency_point.set_pos(
+   ...             self.axis_point,
+   ...             -self.radius*sm.cos(self.origin_angle)*self.parent_axis.cross(self.axis)
+   ...             + self.radius*sm.sin(self.origin_angle)*self.parent_axis,
+   ...         )
+   ...         child_tangency_point.set_pos(
+   ...             self.axis_point,
+   ...             self.radius*sm.cos(self.insertion_angle)*self.child_axis.cross(self.axis)
+   ...             + self.radius*sm.sin(self.insertion_angle)*self.child_axis),
+   ...
+   ...         parent_force_direction_vector = self.origin.pos_from(parent_tangency_point)
+   ...         child_force_direction_vector = self.insertion.pos_from(child_tangency_point)
+   ...         force_on_parent = force_magnitude*parent_force_direction_vector.normalize()
+   ...         force_on_child = force_magnitude*child_force_direction_vector.normalize()
+   ...         loads = [
+   ...             me.Force(self.origin, force_on_parent),
+   ...             me.Force(self.axis_point, -(force_on_parent + force_on_child)),
+   ...             me.Force(self.insertion, force_on_child),
+   ...         ]
+   ...         return loads
+   ...
+
+Also in ``__init__()`` we can calculate some quantities that will be needed in
+multiple overloaded methods::
+
+The length of the pathway is the sum of the lengths of the two linear segments
+and the circular arc that changes with variation of the pin joint coordinate.
+
+The extension velocity is simply the change with respect to time in the arc
+length::
+
+The loads are made up of three forces: two that push an pull on the origin and
+insertion points along the linear portions of the pathway and the resultant
+effect on the elbow from the forces pushing and pulling on the ends of the
+circular arc.
+
+::
+
+
+Now that we have a custom pathway defined we can create a musculotendon
+actuator model in the same fashion as the biceps::
+
+   >>> triceps_pathway = ExtensorPathway(Cm, Dm, P3, B.y, -C.z, D.z, r, q4)
+   >>> triceps_activation = bm.FirstOrderActivationDeGroote2016.with_default_constants('triceps')
+   >>> triceps = bm.MusculotendonDeGroote2016('triceps', triceps_pathway,
+   ...                                        activation_dynamics=triceps_activation)
+   ...
+
+The load formulas are more complex but should allow the triceps to extend the
+elbow::
+
+   >>> # triceps.to_loads()
```

Comment:
I've see both `compute_loads()` and `to_loads()` in some of the new APIs. Is there a reason to have both names? Or can be we consistent?

Reply:
Sam and I just agreed to name them both `to_loads()`.